src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -216,6 +216,11 @@ private function normalizeDefaults(array $defaults): array
             }
         }
 
+        if (!\array_key_exists('stateless', $defaults)) {
+            @trigger_error('Not setting the "api_platform.defaults.stateless" configuration is deprecated since API Platform 2.6 and it will default to `true` in 3.0. You can override this at the operation level if you have stateful operations (highly not recommended).', E_USER_DEPRECATED);
```

Comment:
Why just not default to `false` instead of forcing the user to set this key explicitly?

Reply:
How would you notify that the configuration will default to `true` in `3.0` then?

Reply:
And by the way, it falls back to false already.

WDYT @dunglas ?

Reply:
LGTM!